css/card.css

```diff
@@ -0,0 +1,112 @@
+.card-container {
+    border-radius: 5px;
+    background-color: #ffffffed;
+    width: 23rem;
+    height: 24.3rem;
+    box-shadow: 0px 2px 5px 1px palevioletred;
+}
+.card-container--overlay {
+    width: 23rem;
+    height: 20rem;
+}
+.card-header {
+    display: flex;
+    flex-direction: column;
+}
+.card-img-container {
+    height: 12.8rem;
+}
+.card-img-container > img, 
+.card-img-horizontal > img {
+    max-height: 100%;
+    max-width: 100%;
+}
+.card-title {
+    height: 5rem;
+    margin-left: 1rem;
+    padding-top: 0;
+}
+.title-overlay {
+    position: absolute;
+    z-index: 1;
+    margin-top: 8.5rem;
+    color: white;
+}
+.card-desc {
+    margin: 0rem 1rem 0rem 1rem;
+}
+.card-desc--overlay {
+    margin-top: 1.4rem;
+}
+.card-header--horizontal {
+    flex-direction: row;
+    gap: 5%;
+}
+.card-footer {
+    display: flex;
```

Comment:
suggestion:
If flex is repeated more often in components, You can move flex properties to utility file.

Reply:
yeah it's there in utils. i'll have to refactor all in dev.